### PR TITLE
runners.go: Adding IPAddress field, updated docs url

### PR DESCRIPTION
**Pull request by pmwebster**

### CODE REVIEW

**Comment by pmwebster** [comment]
Sorry, I forgot to rebase :/ Fortunately those are already merged in.

File: runners.go [hunk 1]
```diff
@@ -32,16 +33,17 @@ type RunnersService struct {
 
 // Runner represents a GitLab CI Runner.
 //
-// GitLab API docs: https://docs.gitlab.com/ce/api/runners.html
+// GitLab API docs: https://docs.gitlab.com/ee/api/runners.html
```
Comment (svanharmelen) [inline runners.go]: Why are you updating the URL? I prefer to reference the `ce` docs, unless there are fields or methods in there that are only supported by `ee`.

Comment (svanharmelen) [reply runners.go]: Think I missed that in the other PR, can you refert that?

Comment (pmwebster) [reply runners.go]: sure thing

**Comment by pmwebster** [comment]
The users fields *are* different between CE and EE, given as much, leaving as in. Sorry for the confusion :)

**Comment by pmwebster** [comment]
Would it be easier if I just rebased and make a new PR? I don't know why this is failing.

**Comment by svanharmelen** [comment]
LGTM! Thanks!

**Pull request merged**